### DISCUSSION TITLE
라이브러리에 대한 라이선스 표기가 필요할까요?

### DISCUSSION THREAD

**Discussion by getCurrentThread** [BaekjoonHub/BaekjoonHub#56] [General]
LeetHub에서부터 사용되어오던 jquery, semantic를 포함하여
사용한 모든 라이브러리에 대해 표기가 필요할지 의문이 생겨 토론을 추가합니다.

### 사용된 라이브러리
- jquery 3.3.1 MIT License
- semantic UI 2.4.1 MIT License
- jszip 3.2.0 MIT License *or* the GPLv3 license
- FileSaver 2.0.4 MIT License
- js-sha1 0.6.0 MIT License

### 부가설명
* jszip 라이브러리의 경우에 라이선스를 선택할 수 있는 이중 라이선스입니다. (=양자택일)
* semantic UI 라이브러리의 경우에는 일부 아이콘을 Font Awesome에서 완전 포트하여 활용하고 있습니다. 
(=Font Awesome의 라이선스 제약을 받지 않습니다)

**Comment by flaxinger** [comment]
늦게 확인하여 죄송합니다.
라이선스 표기라면 공식 문서에 명시하는 것을 의미하시는 것 맞을까요?

**Reply by getCurrentThread** [reply]
원래는 별도의 라이센스 페이지 존재한다면 표기하는 것 같습니다. 

현재 모든 라이브러리가 MIT로 활용이 가능해서 표기를 하지 않아도 문제될 사항은 없지만, 
소스코드든 주석으로든 문서화 할 수 있는 아무 곳이나 기록만 해두면 될 것 같습니다.
